assembl/tests/views_tests/test_api2.py

```diff
@@ -987,8 +987,10 @@ def test_add_timeline_event(test_app, discussion):
 class AbstractExport(object):
     # HEADER = {}
 
-    def _get(self, app, discussion_id, lang=None, view_name=None):
+    def _get(self, app, discussion_id, widget_id=0, lang=None, view_name=None, votes_export=False):
```

Comment:
You could simplify by just adding `widget_id` in the end instead of `votes_export` and check your condition against only one variable, instead of two.